site/zenodo_rdm/metrics/utils.py

```diff
@@ -24,7 +24,10 @@ def calculate_metrics(metric_id, cache=True):
     result = deepcopy(current_app.config["METRICS_DATA"][metric_id])
 
     for metric in result:
-        metric["value"] = metric["value"]()
+        try:
+            metric["value"] = metric["value"]()
+        except:
```

Comment:
```suggestion
        except Exception:
```

minor: it's better to specify `Exception` since we don't want e.g. to handle a `SysExit` or `KeyboardInterrupt` (see e.g. [this article](https://www.30secondsofcode.org/python/s/bare-except/))